app/lib/atom_feed.dart

```diff
@@ -23,25 +23,25 @@ class FeedEntry {
   final String alternateTitle;
 
   FeedEntry(this.id, this.title, this.updated, this.authors, this.content,
-            this.alternateUrl, this.alternateTitle);
+      this.alternateUrl, this.alternateTitle);
```

Comment:
There is another place further up, where you replaced a `Configuration.prod() ...;` with `Configuration.prod() ... {}`. But here you don't add an empty clause.

I guess it means if there are field initializes **only then** do we also need the empty block?